.circleci/config.yml

```diff
@@ -62,13 +60,18 @@ commands:
       - run:
           name: 'Install Dependencies'
           command: ./.circleci/install_dependencies.sh
+  setup_vm_browsers:
```

Comment:
nit: Can we call this `install_chrome` instead of `setup_vm_browsers`? (I'm looking to add new steps like `install_firefox` in future if I can get cross-browser tests to work on CircleCI.)

Reply:
Good idea, changing.